src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,14 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] array = new Machine[3];
```

Comment:
let's initialize it in one line and rename this variable
```suggestion
        Machine [] machines = new Machine[] {new Truck(), new Bulldozer(), new Excavator()};
```